### PR TITLE
Jans fido2 attestation mode 9332

### LINKED ISSUE #9332: Replace `skipValidateMdsInAttestationEnabled` with `attestationMode` with values
Replace skipValidateMdsInAttestationEnabled with attestationMode with values:

"disabled" means attestation "none"
"monitor" means attestation "direct" but still accept if none return
"enforced" means that credential creation will fail if attestation is not returned

### PR DESCRIPTION
**Pull request by imran-ishaq**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #9332

#### Implementation Details
  

-------------------
### Test and Document the changes
- [x] Static code analysis has been run locally and issues have been fixed
- [x] Relevant unit and integration tests have been added/updated
- [x] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

Please check the below before submitting your PR. The PR will not be merged if there are no commits that start with `docs:` to indicate documentation changes or if the below checklist is not selected.
- [x] **I confirm that there is no impact on the docs due to the code changes in this PR.**

### CODE REVIEW

File: jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/AppleAttestationProcessor.java [hunk 1]
```diff
@@ -107,23 +103,20 @@ public void process(JsonNode attStmt, AuthData authData, Fido2RegistrationData c
 			// the first certificate in x5c
 			X509Certificate credCert = certificates.get(0);
 
-			if (appConfiguration.getFido2Configuration().isSkipAttestation() ) {
```
Comment (imran-ishaq) [inline jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/AppleAttestationProcessor.java:110]: Just remove the if/else. The inner code is same

File: jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/PackedAttestationProcessor.java [hunk 2]
```diff
@@ -98,20 +95,18 @@ public void process(JsonNode attStmt, AuthData authData, Fido2RegistrationData r
         String signature = commonVerifiers.verifyBase64String(attStmt.get("sig"));
 
         if (attStmt.hasNonNull("x5c")) {
-            if (appConfiguration.getFido2Configuration().isSkipAttestation()) {
```
Comment (imran-ishaq) [inline jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/PackedAttestationProcessor.java:101]: Just remove the if/else. The inner code is same

File: jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/TPMProcessor.java [hunk 3]
```diff
@@ -138,18 +137,16 @@ public void process(JsonNode attStmt, AuthData authData, Fido2RegistrationData c
             List<X509Certificate> trustAnchorCertificates = attestationCertificateService.getAttestationRootCertificates(authData, aikCertificates);
             X509Certificate aikCertificate = aikCertificates.get(0);
 
-            if (appConfiguration.getFido2Configuration().isSkipAttestation()) {
```
Comment (imran-ishaq) [inline jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/TPMProcessor.java:141]: Just remove the if/else. The inner code is same

File: jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/U2FAttestationProcessor.java [hunk 4]
```diff
@@ -115,22 +112,20 @@ public void process(JsonNode attStmt, AuthData authData, Fido2RegistrationData r
             credIdAndCounters.setSignatureAlgorithm(alg);
             List<X509Certificate> trustAnchorCertificates = attestationCertificateService.getAttestationRootCertificates((JsonNode) null, certificates);
 
-            if (appConfiguration.getFido2Configuration().isSkipAttestation()) {
```
Comment (imran-ishaq) [inline jans-fido2/server/src/main/java/io/jans/fido2/service/processor/attestation/U2FAttestationProcessor.java:118]: Just remove the if/else. The inner code is same

**Review by maduvena — APPROVED** [review]

**Pull request merged**